components/org.apache.stratos.manager.console/console/controllers/login/login.jag

```diff
@@ -24,23 +24,29 @@ var log = new Log("controller.login"),
         util = require('/controllers/rest/rest_calls.jag'),
         server = new carbon.server.Server(),
         conf = carbon.server.loadConfig('carbon.xml'),
-        offset = conf.*::['Ports'].*::['Offset'].text(),
-        hostName = conf.*::['HostName'].text().toString();
+        offset = conf. *::['Ports']. *::['Offset'].text(),
+        hostName = conf. *::['HostName'].text().toString(),
+        conf = carbon.server.loadConfig('thrift-client-config.xml'),
+        dasConfig = conf. *::['config'].(name == "das"),
+        dasStatsPublisherEnabled = dasConfig.statsPublisherEnabled.text();
+
 
 if (hostName === null || hostName === '') {
     hostName = 'localhost';
 }
 
 var httpPort = 9763 + parseInt(offset, 10),
-    httpsPort = 9443 + parseInt(offset, 10);
+        httpsPort = 9443 + parseInt(offset, 10);
```

Comment:
It's recommended to declare the vars separately without declaring as single statement.